src/components/styles/PageSection.styled.ts

```diff
@@ -24,21 +24,21 @@ export const TextContainer = styled.div`
   flex-direction: column;
 `
 
-export const Title = styled.div`
+export const Title = styled.div<{ class?: string }>`
   height: 45%;
-  text-align: center;
-  color: #0cbcbc;
+  text-align: ${(props) => (props.class === 'EVENTS' ? 'left' : 'center')};
+  color: ${(props) => (props.class === 'EVENTS' ? 'white' : '#0cbcbc')};
```

Comment:
can help update #0cbcbc to use theme colours?

Reply:
commit : 4b57b73